pkg/rpc/server/server.go

```diff
@@ -1419,15 +1419,15 @@ func (s *Server) submitOracleResponse(ps request.Params) (interface{}, *response
 
 func (s *Server) sendrawtransaction(reqParams request.Params) (interface{}, *response.Error) {
 	if len(reqParams) < 1 {
-		return nil, response.ErrInvalidParams
+		return nil, response.NewInvalidParamsError("not enough parameters", nil)
```

Comment:
Then maybe we can do the same thing for `submitNotaryRequest` and `submitBlock` RPC methods?

Reply:
Makes sense, updated. In general we're already better than C# node with error responses for things like verification failures (thanks to 94430ef3ca1deab4572ba4b959ba5d32e7024996), but these decoding/parameter errors need an improvement.